#277 loading feedback

PaulMicu opened the pull request:
resolves #277

najimgh committed:
#277 merged commit 660ae833803a919a405405354ad73a4f45982bd1

PaulMicu committed:
#277 handle model training feedback

PaulMicu committed:
#277 merged commit from Najim

TalalBazerbachi reviewed (dismissed):
I am getting this error when trying to add an asset.
![image](https://user-images.githubusercontent.com/55160650/114287901-fb4e7080-9a38-11eb-849b-3f5907e3563c.png)
this has nothing to do with this pr, dont use the request changes features to report bugs

PaulMicu committed:
#277 added loading feedback when deleting an asset type

PaulMicu committed:
#277 added loading feedback when deleting an asset type

PaulMicu committed:
#277 merged dev and changed model tab

rdsaliba commented on JavaFXUserInterface/src/main/java/utilities/services/DeleteAssetTypeSercice.java:
```diff
@@ -0,0 +1,45 @@
+
+import static utilities.TextConstants.ASSET_TYPE_LIST_SCENE;
+
+public class DeleteAssetTypeSercice extends Service<String> {
```
There's a typo in the class name. Should be DeleteAssetTypeService

rdsaliba commented on JavaFXUserInterface/src/main/resources/Navigation.fxml:
```diff
@@ -1,37 +1,40 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
```
This file needs to be synchronized with Jeff's last pull request. Should not contain any inline style.
More specifically the textFill and font styling should be removed and use CSS instead. @jeffbuonamici do you see anything else?

rdsaliba reviewed (commented):
Hi Paul. Good job for pulling this one off right on time before the deadline. This brings a great enhancement to the app that was much needed. I'm ok with the coding standards, naming, structure, design. Thank you for doing the unit tests too! I left some comments that requires your attention, should not take too long to fix before merging!

JayChouteau commented on JavaFXUserInterface/src/main/resources/AssetTypeInfo.fxml:
```diff
@@ -6,16 +6,20 @@
+                    AnchorPane.bottomAnchor="0.0"
                     AnchorPane.leftAnchor="59.0" AnchorPane.rightAnchor="0.0" AnchorPane.topAnchor="0.0">
             <children>
+                <ProgressIndicator fx:id="progressIndicator" layoutX="33.0" layoutY="33.0" scaleX="10" scaleY="10"
```
Inline styling. Should be done in the CSS.

JayChouteau reviewed (changes requested):
Nice job on the issue. Really neat and well done changes. I agree with Roy's comments and added 1 myself

JayChouteau committed:
Review comments fixes (#277)
+ Added handling for empty description asset type
- Fxml styling fixing
+ Added handling for empty sliders on model tab
